packages/protocol/contracts/authorizers/SelfAuthorizer.sol

```diff
@@ -15,7 +15,7 @@ contract SelfAuthorizer is ISelfAuthorizer {
   IAirnodeRrp public airnodeRrp;
   mapping(bytes32 => mapping(address => AdminStatus)) public airnodeIdToAdminStatuses;
   mapping(bytes32 => mapping(address => uint256)) public airnodeIdToClientAddressToWhitelistExpiration;
-  mapping(bytes32 => mapping(address => bool)) public airnodeIdToClientAddressToBlacklistStatus;
+  mapping(bytes32 => mapping(address => bool)) public airnodeIdToClientAddressToWhitelistStatus;
```

Comment:
SelfAuthorizer also doesn't need blacklisting anymore, blacklisting should be implemented at the super-admin level